packages/components/src/components/interface/ListViewSimplified/ListViewSimplified.tsx

```diff
@@ -15,7 +15,7 @@ import { StyledPill } from '../Pill'
 
 const Grid = styled.div<{ bottomBorder: boolean }>`
```

Comment:
@rikukissa Yeah they look good to me. Just thinking would it now be possible to merge the mobile and desktop views together as the main reason I made them separate was because of `LabelValueContainer` which we are not using anymore?